### DISCUSSION TITLE
OBS and Bitfocus Companion

### DISCUSSION THREAD

**Discussion by Cadocus** [obsproject/obs-studio#4317] [Development Questions]
Huge thanks to everyone involved in OBS - its a great product and have been using it for a while.  My question is whether it is possible from Bitfocus Companion to pull information from OBS when playing media/videos.  I know if you simply use VLC you can configure a Companion button to show the time length of a clip, the remaining run time, etc.  Is it possible to pull this from OBS in a similar way?
Thanks

**Comment by Fenrirthviti** [comment]
Please use the forums or Discord for support-related questions, thanks!
